### PR TITLE
Adding documentation for attaching a Capture Proxy to a node

### PR DESCRIPTION
**Pull request by okhasawn**
### Description
This PR adds the documentation and all steps required for users to attach a Capture Proxy on a coordinator node.

### Check List
- [ ] New functionality includes testing
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: TrafficCapture/README.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ For more details, check out the [Docker Solution README](dockerSolution/README.m
 
 The Traffic Capture Proxy Server acts as a middleman, capturing traffic going to a source, which can then be used by the Traffic Replayer.
 
+This tool can be attached to coordinator nodes in clusters with a minimum of two coordinator nodes and start capturing traffic
+while having zero downtime on the cluster. More details on attaching a Capture Proxy can be found here: [Capture Proxy](trafficCaptureProxyServer/README.md).
```
Comment (sumobrian) [inline TrafficCapture/README.md:31]: Add a note that the user should be aware of the load on their cluster and make sure that the nodes that remain in service can handle the additional load while nodes are pulled out of service to install the proxy.

File: TrafficCapture/trafficCaptureProxyServer/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+## How to attach a Capture Proxy on a coordinator node.
+
+Follow documentation for [deploying solution](../../deployment/README.md). Then, on a cluster with at least two coordinator nodes, the user can attach a Capture Proxy on a node by following these steps:
+Please note that this is one method for installing the Capture Proxy on a node, and that these steps may vary depending on your environment.
+
+
+These are the **prerequisites** to being able to attach the Capture Proxy:
+
+* **Make sure that the node and your MSK client are in the same VPC and Security Groups**
```
Comment (sumobrian) [inline TrafficCapture/trafficCaptureProxyServer/README.md:11]: Make sure that MSK is accessible by your nodes.

File: TrafficCapture/trafficCaptureProxyServer/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                    "kafka-cluster:WriteData"
+                ],
+                "Resource": "arn:aws:kafka:<region>:<account-id>:topic/migration-msk-cluster-<stage>/*",
+                "Effect": "Allow"
+            }
+        ]
+    }
+  ```
+
+* From **linux command line** of that EC2 instance: Check that the **JAVA_HOME environment variable is set** properly `(echo $JAVA_HOME)`, if not, then try running the following command that might help set it correctly:
```
Comment (sumobrian) [inline TrafficCapture/trafficCaptureProxyServer/README.md:37]: This is not following the same pattern as the other steps. I would start with something like "Verify Java installation is  accessible."

File: TrafficCapture/trafficCaptureProxyServer/README.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+8. **Test the port** that the Capture Proxy is now listening to.
+    1. `curl https://localhost:9200` or `http://`
+    2. You should expect the same response when sending a request to either ports (9200, 19200), except that the traffic sent to the port that the Capture Proxy is listening to, will be captured and sent to your MSK Client, also forwarded to the new Elasticsearch port.
+9. **Verify requests are sent to Kafka**
+   * **Verify that a new topic has been created**
+     1. Log in to the Migration Console container.
+     2. Go the Kafka tools directory
+       cd kafka-tools/kafka/bin
+     3. Create the following file to allow communication with an AWS MSK cluster. Name it: `msk-iam-auth.properties`
```
Comment (sumobrian) [inline TrafficCapture/trafficCaptureProxyServer/README.md:81]: Don't we already have a version of this installed as a helper on the migration console?

**Review by sumobrian — CHANGES_REQUESTED** [review]

File: TrafficCapture/README.md (hunk 1 above)
Comment (okhasawn) [reply TrafficCapture/README.md]: Good idea, thanks Brian!
Addressed in new commit.

File: TrafficCapture/trafficCaptureProxyServer/README.md (hunk 3 above)
Comment (okhasawn) [reply TrafficCapture/trafficCaptureProxyServer/README.md]: Addressed in new commit.

File: TrafficCapture/trafficCaptureProxyServer/README.md (hunk 4 above)
Comment (okhasawn) [reply TrafficCapture/trafficCaptureProxyServer/README.md]: Good catch. I had created that file while testing and forgotten it's already created in one of the directories outside kafka.

**Review by sumobrian — APPROVED** [review]

**Pull request merged**